### ISSUE TITLE
Decommission field `bots_names` in sortinghat section

### ISSUE THREAD

**Issue by sanacl** [chaoss/grimoirelab-sirmordred#369]
The field `bots_names` is no longer needed. The bots are marked via Sortinghat or Hatstall, but at least no instance in Bitergia is using this parameter. IMHO it is not useful at all. So my proposal is to remove the `bots_names` parameter and all its associated code.

```
[sortinghat]
    ...
    bots_names (list: []): Name of the identities to be marked as bots
```

The same should be done with `no_bots_names`

**Comment by valeriocos** [comment]
When you say `The bots are marked via Sortinghat`, do you mean that this is done via identities files or manually on the sortinghat db?

**Comment by sanacl** [comment]
@valeriocos this is currently done in our Bitergia instances via:
* Hatstall (manual)
* Sortinghat (manual)
* Grimoirelab identities file (automatic)

The parameters mentioned in the description are not used to mark bots.

**Comment by valeriocos** [comment]
OK perfect, +1 then.

**Comment by valeriocos** [comment]
Fixed with #373
